README.md

```diff
@@ -1,5 +1,7 @@
 Docker Compose
 ==============
+![Docker Compose](https://github.com/docker/compose/blob/master/docs/logo.png).
```

Comment:
I don't think this url will work, doesn't it need to be something like `https://raw.githubusercontent.com/docker/compose/master/docs/logo.png` ?